core/include/initcall.h

```diff
@@ -139,6 +137,8 @@ struct initcall {
 #define nex_release_init_resource(fn)	release_init_resource(fn)
 #endif
 
+#define boot_final(fn)			__define_initcall(final, 8, fn)
```

Comment:
Maybe swap order in inline comment at line 83:
s/`boot_final() / nex_*init*()`/`nex_*ini / boot_final()`/

I think there should be oter some updates in the inline description because some sentences are wrong or not accurate, e.g.:
*"preinit_*() are always called before functions registered with *_init()."*
*"boot_final() functions are called first before exiting to normal world the first"*

Reply:
I'll swap boot_final() / nex_*init*() in the comment above.
I'll propose an update in a later PR for the description.